website/migrations/0253_alter_organization_url.py

```
@@ -0,0 +1,17 @@
+# Generated by Django 5.1.8 on 2025-11-20 18:28
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("website", "0252_fix_organization_logo_field_length"),
+    ]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)